frontend/src/components/Home.vue

```diff
@@ -162,6 +186,10 @@ export default {
   },
   data() {
     return {
+      authRoutes: AuthRoutes,
+      KeycloakCommonServiceList: CommonServiceList.filter(function(x) {
```

Comment:
`CommonServiceList.filter(x => {`
This function doesn't depend on `this` scope so it can be declared anonymously

Reply:
good call.. .this view does look quite bland. apparently it will be getting more content soon.